hera_mc/cm_hookup.py

```diff
@@ -277,7 +282,18 @@ def __header_entry_name_adjust(self, col):
             colhead = col
         return colhead
 
-    def show_hookup(self, hookup_dict, cols_to_show, show_levels):
+    def get_correlator_input_from_hookup(self, hookup_dict, corr_name=PC.roach_input_name):
+        """
+        Retrieve the correlator inputs from a hookup_dictionary
+        """
+        corr_input = {}
+        if corr_name in hookup_dict['columns']:
```

Comment:
So, if `corr_name` is not in `hookup_dict['columns']`, this function will return `{}`. Are those the best semantics? Would it be better to raise an exception in this case?